sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -6,152 +6,28 @@ export const isNode =
 
 /**
  * Enum to abstract away string values used for the Environment Variable key names.
- * These are the entity name related keys to be configured for Node platform and
- * get suffixed with `-BROWSER` in context of browser platform.
- * This is done in order to allow for browser and node tests to be
- * run in parallel by using independent entities.
  */
 export enum EnvVarKeys {
   SERVICEBUS_CONNECTION_STRING = "SERVICEBUS_CONNECTION_STRING",
   AAD_CLIENT_ID = "AAD_CLIENT_ID",
   AAD_CLIENT_SECRET = "AAD_CLIENT_SECRET",
-  AAD_TENANT_ID = "AAD_TENANT_ID",
-
-  QUEUE_NAME = "QUEUE_NAME",
-  QUEUE_NAME_NO_PARTITION = "QUEUE_NAME_NO_PARTITION",
-  QUEUE_NAME_SESSION = "QUEUE_NAME_SESSION",
-  QUEUE_NAME_NO_PARTITION_SESSION = "QUEUE_NAME_NO_PARTITION_SESSION",
-  TOPIC_NAME = "TOPIC_NAME",
-  TOPIC_NAME_NO_PARTITION = "TOPIC_NAME_NO_PARTITION",
-  TOPIC_NAME_SESSION = "TOPIC_NAME_SESSION",
-  TOPIC_NAME_NO_PARTITION_SESSION = "TOPIC_NAME_NO_PARTITION_SESSION",
-  SUBSCRIPTION_NAME = "SUBSCRIPTION_NAME",
-  SUBSCRIPTION_NAME_NO_PARTITION = "SUBSCRIPTION_NAME_NO_PARTITION",
-  SUBSCRIPTION_NAME_SESSION = "SUBSCRIPTION_NAME_SESSION",
-  SUBSCRIPTION_NAME_NO_PARTITION_SESSION = "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
-  TOPIC_FILTER_NAME = "TOPIC_FILTER_NAME",
-  TOPIC_FILTER_SUBSCRIPTION_NAME = "TOPIC_FILTER_SUBSCRIPTION_NAME",
-  TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME = "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
-  MANAGEMENT_QUEUE_1 = "MANAGEMENT_QUEUE_1",
-  MANAGEMENT_TOPIC_1 = "MANAGEMENT_TOPIC_1",
-  MANAGEMENT_SUBSCRIPTION_1 = "MANAGEMENT_SUBSCRIPTION_1",
-  MANAGEMENT_RULE_1 = "MANAGEMENT_RULE_1",
-  MANAGEMENT_QUEUE_2 = "MANAGEMENT_QUEUE_2",
-  MANAGEMENT_TOPIC_2 = "MANAGEMENT_TOPIC_2",
-  MANAGEMENT_SUBSCRIPTION_2 = "MANAGEMENT_SUBSCRIPTION_2",
-  MANAGEMENT_RULE_2 = "MANAGEMENT_RULE_2",
-  MANAGEMENT_NEW_ENTITY_1 = "MANAGEMENT_NEW_ENTITY_1",
-  MANAGEMENT_NEW_ENTITY_2 = "MANAGEMENT_NEW_ENTITY_2"
```

Comment:
The karma.conf.js file needs to be updated to remove the above env vars as well

Reply:
Updated